pages/contact/index.tsx

```diff
@@ -1,5 +1,107 @@
+import {
+  Box,
+  FormControl,
+  FormLabel,
+  Heading,
+  Input,
+  NumberInput,
+  NumberInputField,
+  Text,
+  Flex,
+  Button,
+} from '@chakra-ui/react';
+import { useState } from 'react';
+
 const ContactUs: React.FC = () => {
-  return <>This is the Contact Us page</>;
+  const [firstName, setFirstName] = useState<string>('');
+  const [lastName, setLastName] = useState<string>('');
+  const [email, setEmail] = useState<string>('');
+  const [zipCode, setZipCode] = useState<string>();
+
+  return (
+    <>
+      <Box margin="auto" maxWidth="80%" marginTop="50px">
+        <Heading>We would love to hear from you and keep you updated with our progress!</Heading>
+        <Box margin="50px 0">
+          <Box margin="25px 0">
+            <Text>
+              For questions, press inquiries, or other issues please contact us at{' '}
+              <b>
+                <a href="mailto:info@outstandinglife.org">info@outstandinglife.org</a>
+              </b>
+              .
+            </Text>
+          </Box>
+          {/* hiding form for now because we don't have registration functionality */}
+          {false && (
```

Comment:
💭 This is great! Hiding unfinished code behind a false value is similar to "feature flagging", and we ought to do it here!

If you had a `featureFlags.ts` file, exporting a boolean, you could change this line to:
```
CONTACT_FORM_FLAG && ...
```

Reply:
⛏️ Forms ought to be their own component

Reply:
agreed with both

Reply:
Addressed both of these comments :​)